app/server/api/ussd_api.py

```diff
@@ -31,19 +32,24 @@ def post(self):
         user_input = post_data.get('text')
         service_code = post_data.get('serviceCode')
 
+
         if phone_number:
             msisdn = proccess_phone_number(phone_number, 'KE')
-            user = User.query.filter_by(phone=msisdn).first()
+            user = User.query.execution_options(show_all=True).filter_by(phone=msisdn).first()
```

Comment:
need this since the API doesn't have an org...? user wasn't being found. is this the correct way to do this or how to do?

Reply:
this is a totally fine way of doing it, though you can also use the `show_all` decorator if you're absolutely confident that the information being queried only relates to the user. We use this on the me api:
```
from server.utils.auth import requires_auth, show_all


class MeAPI(MethodView):
    @requires_auth
    @show_all
    def get(self):

```